weekly-updates/Maria-Eriksson/Weekly-meetings.md

```diff
@@ -14,6 +15,58 @@
 * [7 September 2020](#date-7-september-2020)
 * [12 August 2020](#date-12-august-2020)
 
+
+### Date: 4 February 2021
+
+#### Who did you help this week?
+
+* BSc student about neuropsychology project
+* GA's collaborator (KW offered helpful advice - thank you!!)
+* Chat with neurosurgeon about neuropsychology
+* Partner
+  
+#### Who helped you this week?
+
+* Data buds chats and co-working
+
+#### What did you achieve?
+
+* Completed RSE course
+* Finished retrieving seizure and AED outcome for ALL PATIENTS OMG
+* Read commentary by EP & DJ
+* Worked on cross-validation script (but still stuck)
+* Had first official mentoring session with Isla, which was lovely
+* Follow-up re XNAT
+* Notes on data standards & open and reproduciblity practices in epilepsy surgery research - potential commentary paper
+
+#### What did you struggle with?
+
+* Month 2 of being in Finland, and am missing my London life (which I realise must sound bizarre considering the current situation) and my partner
+
+#### What would you like to work on next week?
+
+* Time to properly get project on GitHub (though through smaller steps, under a 'top level issue')
+	* Preliminary plan is to have 3 (private, for now) repositories: 1) paper on data standards & open and reproduciblity practices in epilepsy surgery research, 2) paper on "epilepsy surgery in children from 2000-2018: a single centre experience" (but need to get KW's thoughts on this and need to look into whether releasing the data in this paper is possible), and 3) predictive model scripts
```

Comment:
This all looks great - you can still do a lot of the project management for the work in GitHub - its a super helpful exercise to practice coordinating the work and planning the different steps.... even if you can't end up making public versions of the repositories.

Strong support on the private to start off with. But do add folks liberally so they can help you 💖